filter_dom.py

```
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+from bs4 import BeautifulSoup
+
+
+def extract_external_js(html_content):
```

Ruff (0.8.2)

Missing return type annotation for public function `extract_external_js`

(ANN201)